mk/Common.mk

```diff
@@ -243,15 +243,10 @@ ASM_DIR   ?= $(ASM)
 #
 # Note that the DSIM targets allow for writing the log-files to arbitrary
 # locations, so all of these paths are absolute, except those used by Verilator.
-# TODO: clean this mess up!
 CORE_TEST_DIR                        = $(CORE_V_VERIF)/$(CV_CORE_LC)/tests/programs
 BSP                                  = $(CORE_V_VERIF)/$(CV_CORE_LC)/bsp
 FIRMWARE                             = $(CORE_TEST_DIR)/firmware
 VERI_FIRMWARE                        = ../../tests/core/firmware
-CUSTOM                               = $(CORE_TEST_DIR)/custom
```

Comment:
I'll accept this (because its the right thing to do), but will need to check close to see if it causes problems for the CORE testbench.   Leave it to me.